tailwind.config.ts

```diff
@@ -344,6 +348,9 @@ const config: Config = {
         ".hyphens-none": { hyphens: "none" },
       })
     },
+    function ({ addVariant }: { addVariant: (name: string, generator: string) => void }) {
+      addVariant("not-first", "&:not(:first-child)")
+    },
```

Comment:
What does this code enable?

Reply:
![image](https://github.com/user-attachments/assets/f2349de7-16fa-473c-a31a-5d0b6bc4d637)

Reply:
It's the other way around - we need the margin-top to be there on every `<hr>` that's not the first `<hr>`

Reply:
Let me show you what I mean tomorrow :)